fe/fe-core/src/main/java/com/starrocks/planner/PlanFragment.java

```diff
@@ -194,6 +194,18 @@ public ExchangeNode getDestNode() {
         return destNode;
     }
 
+    public List<PlanNode> getOlapScanNodes(PlanNode root) {
+        if (root instanceof OlapScanNode) {
+            return Lists.newArrayList(root);
+        }
+
+        List<PlanNode> scanNodes = Lists.newArrayList();
```

Comment:
The  `List<PlanNode> scanNodes` could pass as function args, then, we only need create one array list, not many.

Reply:
ok